applications/nrf5340_audio/src/modules/sd_card_playback.c

```diff
@@ -0,0 +1,567 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "sd_card_playback.h"
+
+#include <stdint.h>
+#include <math.h>
+#include <zephyr/sys/ring_buffer.h>
+#include <zephyr/shell/shell.h>
+
+#include "sd_card.h"
+#include "sw_codec_lc3.h"
+#include "sw_codec_select.h"
+#include "pcm_mix.h"
+#include "audio_system.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(sd_card_playback, CONFIG_MODULE_SD_CARD_PLAYBACK_LOG_LEVEL);
+
+#define MAX_PATH_LEN	    (CONFIG_FS_FATFS_MAX_LFN)
+#define LIST_FILES_BUF_SIZE 512
+#define FRAME_DURATION_MS   (CONFIG_AUDIO_FRAME_DURATION_US / 1000)
+
+#define WAV_FORMAT_PCM	    1
+#define WAV_SAMPLE_RATE_48K 48000
+
+#pragma pack(1)
```

Comment:
Any particular reason for wanting to override the compiler here? Same for the pragma below. This might cause the speed to decrease, and I don't necessarily see why we would need to alter the alignment of these structs

Reply:
Talked to Andreas and was told it was to easier read out the header of lc3/wav. Suggest using __packed instead, see le_audio.rx.h for an example